prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -125,17 +127,27 @@ else
   if [ $BEAMTYPE == "PbPb" ]; then
     [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
   elif [ $BEAMTYPE == "pp" ]; then
-    ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=1;fastMultConfig.cutMultClusHigh=2000;fastMultConfig.cutMultVtxHigh=500;ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2"
-    [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
+    if [ -z $ISPILOTDATA ]; then # configuration of ITS tracking for pilot beam data
+      ITS_CONFIG_KEY+="ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2;ITSCATrackerParam.useDiamond=true;ITSCATrackerParam.diamondPos[2]=35.;ITSCATrackerParam.pvRes=10."
+      [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode sync_misaligned"
+    else
+      ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=1;fastMultConfig.cutMultClusHigh=2000;fastMultConfig.cutMultVtxHigh=500;ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2"
+      [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
+    fi
   elif [ $BEAMTYPE == "cosmic" ]; then
     [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode cosmics"
   else
-    [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
+    [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode sync_misaligned"
```

Comment:
all these parameters can be set via env. vars, as it was done during the pilot beam, why do we need to hardcode them?

Reply:
We adapted to the script, also so to reduce to the minimum the env variables that we pass, which is error prone.

Reply:
BTW, the syntax mean that if they are already there, what is defined is taken before the hardcoded values, so it is already what yo suggest.

Reply:
But you can define these extra params in a separate script which calls dpl-workflow. It is much cleaner than patching general script by custom settings. Also, I am sure the custom parameters for the prim.vertexin and TPC/ITS matching we will use in the async pass will not be the same quick and dirty settings used in the sync. reco.

Reply:
Yes, an extra script for env variables is what I am planning to have anyway - we will need it e.g. to define which detectors are in the workflow and other things. But if we can get a default of the settings, isn't it practical to have them directly in the script? 
I am aware that settings will change, so we should always set the parameters only if they are not set from outside. I will check and fix where it is not done.

Reply:
Well, the dpl-workflow is a general purpose script, I don't see what we gain by hardcoding the custom pilot beam option in it instead of defining the same env. var in the external steering script. But we certainly increase its complexity by putting there custom settings. Also, I am not sure we will use the same settings for different periods of this data...

Reply:
Done.